### ISSUE TITLE
Formatting in the screenplay editor

### ISSUE THREAD

**Issue by tiamottu** [dimkanovikov/starc#17]
In STARC v. 0.0.4 formatting one or more words in the screenplay editor (the panel named "Text") as italic, bold or underlined causes all the words before these to be formatted too. When this new formatting is removed in the first selection, some of the words that have been influenced still are in italic, bold or underlined.
I don't know if this is a known bug and if its correction is already planned, but I still thought it might have been a good idea to write about it.

**Comment by dimkanovikov** [comment]
Hello, my friend. Not sure that I correctly understand what you mean. Can you please give me step by step guide of how to reproduce this bug?

**Comment by tiamottu** [comment]
In the Linux version of v. 0.0.4 free, after creating a screenplay and a document inside the screenplay, go to Nameofdocument>Text (I assume it is what in KIT Scenarist was called Script). If you then write something (I tried with a standard scene heading, action and some dialogue) and format it (I made one of the words in the dialogue italic), every other word before it will be formatted too (in my case every word before that, including the scene heading becomes italic).
